superset-frontend/src/components/Select/styles.tsx

```diff
@@ -257,12 +257,14 @@ export const DEFAULT_STYLES: PartialStylesConfig = {
   ],
 };
 
-const inputTagStyles = {
-  background: 'none',
-  border: 'none',
-  outline: 'none',
-  padding: 0,
-  width: '100%',
+const inputTagStyles = (isMultiWithValue: boolean | undefined) => {
+  const styles = {
+    background: 'none',
+    border: 'none',
+    outline: 'none',
+    padding: 0,
+  };
```

Comment:
perhaps this object should be defined as a const as INPUT_TAG_STYLES?

Then you could get rid of the function entirely and just do:
```tsx
inputStyle={isMultiWithValue ? { ...INPUT_TAG_STYLES, width: '100%' } : INPUT_TAG_STYLES}
```
on line 335

Reply:
SGTM! I made one slight change, and named it `INPUT_TAG_BASE_STYLES` b/c they don't apply as is to all the input tags.